### PR TITLE
fix #131, fix #132. rewrite ending of abstract. fix intro figure

### LINKED ISSUE #131: Remove/rewrite abstract ending

### LINKED ISSUE #132: Improve introduction figure

### PR DESCRIPTION
**Pull request by SebastianMygind**

### CODE REVIEW

File: src/introduction/introduction.tex [hunk 1]
```diff
@@ -2,10 +2,10 @@
 \chapter{Introduction}\label{ch:introduction}
 % textidote: ignore end
 
-Chess has seen a rise in player count in recent years, and with the growing player base the current chess websites have
+Chess has seen a rise in player count in recent years, and with the growing player base, the current chess websites have
 perfected their craft.
 This has been done through intuitive design and the addition of extra features in the apps.
-One reason that chess has become more popular online is that using popular web apps like \url{chess.com} or
+One reason that chess has become more popular online is that using popular web apps like \url{chess.com}or
```
Comment (audio-engineer) [inline src/introduction/introduction.tex]: This cannot be correct, right?

File: src/introduction/introduction.tex [hunk 2]
```diff
@@ -14,33 +14,35 @@ \chapter{Introduction}\label{ch:introduction}
...
+    Chess engine coaching exists online in the cloud, no physical meeting is necessary.}\label{fig:project-overview}
 \end{figure}
 % textidote: ignore end
 
-Chess engine popularity has risen greatly in the past years and have been developed to a point that they
+Chess engine popularity has risen greatly in the past years and has been developed to a point that they
 outperform humans.
-These tools are essential to understanding the complexity of the end- and mid-game strategies of chess.
-This makes engines an optimal tool for teaching players the intricacies of chess which is what
+These tools are essential to understanding the complexity of mid/end-game strategies of chess.
```
Comment (audio-engineer) [inline src/introduction/introduction.tex]:
```suggestion
These tools are essential to understanding the complexity of mid- and endgame strategies of chess.
```

**Review by audio-engineer — DISMISSED** [review]

File: src/main.tex [hunk 3]
```diff
@@ -64,8 +64,13 @@
 The key focus points of the report are the learning aspect of chess, primarily focusing on why assisted
 learning, such as coaching, is more effective than individual learning.
 The outcome is an online multiplayer game, where two people can play against each other, while
-receiving coaching from the engine in the form of feedback on the players' moves, and suggest improvements.
-The project concludes with a discussion on how AI can be used to improve the chess learning experience}
+receiving coaching from a chess engine in the form of feedback on the players' moves, and suggest improvements.
+The results the authors found during this project indicate that using chess engines for coaching is viable, but it is
+lacking in terms of communicating with the players.
+A promising addition to chess engine coaching is are LLMs, as they could make up for chess engines'
```
Comment (Danar435) [inline src/main.tex]: Spelling mistake, remove 'is'

**Review by Danar435 — CHANGES_REQUESTED** [review]

File: src/main.tex (hunk 3 above)
Comment (TexWO) [reply src/main.tex]: line 70 btw

File: src/introduction/introduction.tex (hunk 2 above)
Comment (Danar435) [reply src/introduction/introduction.tex]: I think that "mid- to endgame" sounds better

**Review by TexWO — APPROVED** [review]

**Review by Danar435 — APPROVED** [review]

**Comment by SebastianMygind** [comment]
@audio-engineer I have implemented your changes will dismiss review to not block merge.

**Pull request merged**